cmd/orb.go

```diff
@@ -278,7 +284,7 @@ func promoteOrb(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	Logger.Infof("Orb promoted to %s", response.Orb.Version)
+	Logger.Infof("Orb %s/%s@%s promoted to %s", args[0], args[1], devLabel(args[2]), response.Orb.Version)
```

Comment:
I think all these string formatting changes in `orb.go` are a good opportunity to isolate API response details into `api.go` and make our data structures (Orb, OrbVersion) return just our domain concept details. In these cases, they could return an Orb reference and Orb Version reference.